### PR TITLE
Features/update unit tests for sources

### PR DESCRIPTION
**Pull request by SamirHassoun**
1) ClearDocumentRequestFromQueueCommandHandler: Code tidy
2) ClearDocumentRequestFromQueueCommand: Removed unused property
3) EditDatabaseTests: Added test for Session filename is generated with timestamp.
4) SessionFileGeneratorTests: Added test for Sources being added to Session file
5) Added unit test for ClearDocumentRequestFromQueueCommandHandler
6) Use System.IO.Abstractions and System.IO.Abstractions.TestingHelpers in SourceDocumentCoordinator to assists in unit tests
7) Added unit test for PersistDocumentInStoreCommandHandlerTests

### CODE REVIEW

File: src/SourceDocumentCoordinator/SourceDocumentCoordinator.UnitTests/PersistDocumentInStoreCommandHandlerTests.cs [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+            var dbContextOptions = new DbContextOptionsBuilder<WorkflowDbContext>()
+                .UseInMemoryDatabase(databaseName: "inmemory")
+                .Options;
+
+            _dbContext = new WorkflowDbContext(dbContextOptions);
+
+            _fakeContentServiceApiClient = A.Fake<IContentServiceApiClient>();
+            _fakeDocumentStatusFactory = A.Fake<IDocumentStatusFactory>();
+            _fakeDocumentFileLocationFactory = A.Fake<IDocumentFileLocationFactory>();
+            _fakeFileSystem = A.Fake<IFileSystem>();
```
Comment (dynamictulip) [inline src/SourceDocumentCoordinator/SourceDocumentCoordinator.UnitTests/PersistDocumentInStoreCommandHandlerTests.cs:44]: You _can_ use fakeiteasy on the file system but I'd strongly recommend including the `System.IO.Abstractions.TestingHelpers` nuget package and then using the dedicated `MockFileSystem`

File: src/SourceDocumentCoordinator/SourceDocumentCoordinator.UnitTests/PersistDocumentInStoreCommandHandlerTests.cs [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+            var message = new PersistDocumentInStoreCommand()
+            {
+                CorrelationId = Guid.NewGuid(),
+                ProcessId = 5678,
+                SourceDocumentId = 1999999,
+                SourceType = SourceType.Database,
+                Filepath = "DoesNotExists.tif"
+            };
+
+            A.CallTo(() => _fakeFileSystem.File.ReadAllBytes(message.Filepath)).Throws<FileNotFoundException>();
```
Comment (dynamictulip) [inline src/SourceDocumentCoordinator/SourceDocumentCoordinator.UnitTests/PersistDocumentInStoreCommandHandlerTests.cs:192]: With a `MockFileSystem` this line will no longer be needed

Comment (SamirHassoun) [reply src/SourceDocumentCoordinator/SourceDocumentCoordinator.UnitTests/PersistDocumentInStoreCommandHandlerTests.cs]: fixed, thanks

File: src/SourceDocumentCoordinator/SourceDocumentCoordinator.UnitTests/PersistDocumentInStoreCommandHandlerTests.cs (hunk 1 above)
Comment (SamirHassoun) [reply src/SourceDocumentCoordinator/SourceDocumentCoordinator.UnitTests/PersistDocumentInStoreCommandHandlerTests.cs]: Fixed, thanks

**Pull request merged**